src/session.rs

```diff
@@ -162,6 +166,13 @@ pub fn create_prepared_batch(queries: Vec<&PreparedStatementWrapper>) -> BatchWr
     BatchWrapper { inner: batch }
 }
 
+fn configure_session_builder(options: &SessionOptions) -> SessionBuilder {
+    let mut builder = SessionBuilder::new();
+    builder = builder.custom_identity(get_self_identity(options));
```

Comment:
:pick: It really pains me when I see naming of `get_*` when not really referring to just retrieving a field. We should rather use `create_*`, `build_*`, `configure_*` etc., whichever you find best.

Reply:
The same goes for `getWrappedType`. IMO, it should be `wrapType`, because the wrapped type has not existed anywhere yet -- we are creating it on-fly.